client/a8c-for-agencies/components/sidebar-menu/hooks/use-main-menu-items.tsx

```diff
@@ -169,10 +169,11 @@ const useMainMenuItems = ( path: string ) => {
 							icon: commentAuthorAvatar,
 							path: '/dashboard',
 							link: A4A_PARTNER_DIRECTORY_DASHBOARD_LINK,
-							title: translate( 'Partner Directory' ),
+							title: translate( 'Partner Directories' ),
 							trackEventProps: {
-								menu_item: 'Automattic for Agencies / Partner Directory',
+								menu_item: 'Automattic for Agencies / Partner Directories',
```

Comment:
I feel like we should keep this as-is.

Reply:
Hmm, I'm not sure if changing this has any implications, but to reflect the correct language we use in the UI, I think it’s fine to make this change.. 🤔

Reply:
Good point. I've reverted it back.. 👍